desktop/core/src/desktop/models.py

```diff
@@ -1808,7 +1806,8 @@ def get_config(self):
       'has_computes': self.cluster_type in ('cdw', 'altus', 'snowball'),  # or any grouped engine connectors
       'hue_config': {
         'enable_sharing': ENABLE_SHARING.get(),
-        'collect_usage': COLLECT_USAGE.get()
+        'collect_usage': COLLECT_USAGE.get(),
+        'allow_sample_data_from_views': ALLOW_SAMPLE_DATA_FROM_VIEWS.get(),
```

Comment:
Why are we adding it here when we have already added it in the /get_config API?
Looks good to remove from here?

Reply:
The "hue_config" is a property of the main config object were we put newer flags. The parent (main) config object is cluttered with legacy stuff.

Reply:
I will shift this whole section in /get_config for the time being. For a long term solution, we should clean up our config handling and its APIs